extensions/wikia/InsightsV2/specials/PopularPages.class.php

```diff
@@ -73,12 +73,12 @@ public function recache( $limit = false, $ignoreErrors = true ) {
 	 * @return array
 	 */
 	public function reallyDoQuery( $limit = false, $offset = false ) {
-		global $wgCityId;
+		global $wgCityId, $wgContentNamespaces;
 
 		$topArticles = DataMartService::getTopArticlesByPageview(
 			$wgCityId,
 			null,
-			null,
+			$wgContentNamespaces,
```

Comment:
@idradm It should. By default `$wgContentNamespaces` has value of `array( NS_MAIN );`